src/main/java/org/jabref/gui/exporter/ExportAction.java

```diff
@@ -82,7 +82,7 @@ public void actionPerformed(ActionEvent e) {
 
                     ExportFileFilter eff = (ExportFileFilter) ff;
                     String path = file.getPath();
-                    if (!path.endsWith(eff.getExtension())) {
+                    if (!path.endsWith(eff.getExtension().toString())) {
```

Comment:
This will not work: the `FileExtensions.toString` method will return the default java string serialization and not the extension stored internally. You should use `FileExtensions.getExtensionsAsList()` instead.